healthcare/api-client/datasets/datasets.py

```diff
@@ -33,9 +33,9 @@ def get_client(service_account_json, api_key):
         service_account_json)
     scoped_credentials = credentials.with_scopes(api_scopes)
 
-    discovery_url = '{}?labels=CHC_ALPHA&version={}&key={}'.format(
+    discovery_url = '{}?labels=CHC_BETA&version={}&key={}'.format(
         discovery_api, api_version, api_key)
-
+    
```

Comment:
nit. trailing whitespace

Reply:
I just fixed the two lint errors. I'll remove the trailing whitespace ASAP.